src/worker/worker_template.py

```diff
@@ -97,7 +94,10 @@ def _get_and_write_metrics(self, data, model_output):
         acc_metrics = np.zeros(len(self.evaluation_metrics))
         for i, metric in enumerate(self.evaluation_metrics):
             acc_metrics[i] += metric(data, model_output)
-            self.writer.add_scalar(metric.nickname, acc_metrics[i])
+            try:
+                self.writer.add_scalar(metric.nickname, acc_metrics[i])
+            except AttributeError:
+                pass
```

Comment:
When will `AttributeError` be raised?

Reply:
See the comments

Reply:
Decouple `_get_metrics()` and `_write_metrics()` and overwrite _write_metrics in the evaliuator would be a better way, imo.